connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewContainerController.kt

```diff
@@ -110,8 +165,13 @@ internal class StripeConnectWebViewContainerController<Listener : StripeEmbedded
     fun shouldOverrideUrlLoading(context: Context, request: WebResourceRequest): Boolean {
         val url = request.url
         return if (url.host?.lowercase() in ALLOWLISTED_HOSTS) {
-            // TODO - add an analytic event here to track this unexpected behavior
             logger.warning("($loggerTag) Received pop-up for allow-listed host: $url")
+            analyticsService.track(
+                ConnectAnalyticsEvent.ClientError(
+                    error = "Unexpected pop-up",
```

Comment:
IMO `error_code` would be better than a verbose `error`.

Reply:
➕ Related to my comment above, I think giving it a name like `error_code` means it's less likely we'd inadvertently include an error message that contained PII.